Deprecate package metadata, improve packaging documentation, refactor record package schema (#1640)
open-contracting/standard: docs/schema/packaging/index.md

# Packaging

> In this section you will find the schema for [release packages](release_package.md) and [record packages](record_package.md), which act as envelopes for releases and records, respectively.

Reviewer: ```suggestion
This section describes the schema for [release packages](release_package) and [record packages](record_package), which act as envelopes for releases and records, respectively.
```
